core/services/synchronization/telemetry_ingress_batch_worker.go

```diff
@@ -4,10 +4,11 @@ import (
 	"sync"
 	"time"
 
+	"go.uber.org/atomic"
```

Comment:
We just switched all of these to the std lib in #8345

Reply:
good to know, will update the PR